go/apps/api/routes/v2_keys_get_key/handler.go

```
@@ -98,261 +97,211 @@ func (h *Handler) Handle(ctx context.Context, s *zen.Session) error {
-		Start:       key.Start,
-		Plaintext:   plaintext,
-		Name:        nil,
-		Meta:        nil,
-		Identity:    nil,
-		Credits:     nil,
-		Expires:     nil,
-		Permissions: nil,
-		Ratelimits:  nil,
-		Roles:       nil,
-		UpdatedAt:   nil,
+	response := openapi.KeyResponseData{
```

golangci-lint (2.2.2)

openapi.KeyResponseData is missing fields Credits, Expires, Identity, Meta, Name, Permissions, Ratelimits, Roles, UpdatedAt

(exhaustruct)